modules/tinymce/src/plugins/help/test/ts/browser/PluginTabsOrderTest.ts

```diff
@@ -0,0 +1,63 @@
+import { Keyboard, Keys, Mouse, UiFinder } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { SelectorFilter, SugarBody, SugarDocument, TextContent } from '@ephox/sugar';
+import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import HelpPlugin from 'tinymce/plugins/help/Plugin';
+
+import { selectors } from '../module/Selectors';
+import FakePlugin from '../module/test/FakePlugin';
+import NoMetaFakePlugin from '../module/test/NoMetaFakePlugin';
+
+describe('Browser Test: .PluginTabsOrderTest', () => {
```

Comment:
```suggestion
describe('browser.tinymce.plugins.help.PluginTabsOrderTest', () => {
```
TinyMCE tests use this format for the name 🤷

Reply:
The one I used as a basis (Metadata) was the *only* test that didn't do this. I'll update it too.

Reply:
ahh damn thanks for that! We did try to fix all of them a while back but must have missed that.